### PR TITLE
[backend-api] backend api is now put under review for future changes

### PR DESCRIPTION
**Pull request by grossmannmartin**
- package is now not tested with the new versions

| Q             | A
| ------------- | ---
|Description, reason for the PR| Current state of the Backend API does not comply with our vision of the product. The package has been put under review.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| Yes 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -121,6 +118,7 @@
         "psr/log": "^1.0",
         "ramsey/uuid": "^3.8",
         "roave/better-reflection": "^3.5",
+        "sensio/framework-extra-bundle": "^5.2",
```
Comment (vitek-rostislav) [inline composer.json:121]: why is this package added, please?

**Review by vitek-rostislav — APPROVED** [review]

**Comment by vitek-rostislav** [comment]
I am thinking, should not we mention this in the upgrade notes? :thinking: If anyone used the BE API, they would probably need to fork it and deal with PHP 8.1 support themselves during the upgrade to the new version of the framework.

**Review by vitek-rostislav — CHANGES_REQUESTED** [review]
I am thinking, should not we mention this in the upgrade notes? :thinking: If anyone used the BE API, they would probably need to fork it and deal with PHP 8.1 support themselves during the upgrade to the new version of the framework.

File: composer.json (hunk 1 above)
Comment (grossmannmartin) [reply composer.json:121]: It's used in project-base and the dependency was in monorepo installed as a transitive dependency with now removed fos/rest-bundle

**Comment by grossmannmartin** [comment]
> I am thinking, should not we mention this in the upgrade notes? 🤔 If anyone used the BE API, they would probably need to fork it and deal with PHP 8.1 support themselves during the upgrade to the new version of the framework.

Is it ok? 131e20049

**Pull request merged**